arangod/Cluster/ClusterInfo.cpp

```diff
@@ -804,7 +801,6 @@ void ClusterInfo::loadPlan() {
               << ". invalid information in Plan. The view will "
               << "be ignored for now and the invalid information will "
               << "be repaired. VelocyPack: " << viewSlice.toJson();
-
           TRI_ASSERT(false);
```

Comment:
@kvahed @KVS85 can we remove this assert?

Reply:
@fceller there are many similar assertions in `loadPlan` and it was like this before.